tests/unit/test_cmr_search.py

```diff
@@ -18,193 +18,230 @@
 
 
 class TestQuery(TestCase):
-    ''' A class for testing functions in cmr_search.
-    '''
+    ''' A class for testing functions in cmr_search. '''
     @classmethod
+    def setUpClass(cls):
```

Comment:
Really like this! It get's a little annoying retyping this for all the tests

Reply:
:100:

I love these methods on `TestCase`. There are times when they save _soooo_ many lines of code.